src/SignalR/clients/ts/signalr/src/HandshakeProtocol.ts

```diff
@@ -40,7 +40,20 @@ export class HandshakeProtocol {
             // optional content after is additional messages
             const responseLength = separatorIndex + 1;
             messageData = String.fromCharCode.apply(null, binaryData.slice(0, responseLength));
-            remainingData = (binaryData.byteLength > responseLength) ? binaryData.slice(responseLength).buffer : null;
+
+            if (binaryData.byteLength > responseLength) {
+                remainingData = binaryData.slice(responseLength);
+                // IE 11 does not have .slice() on Uint8Array, we polyfill it with Array.prototype.slice()
```

Comment:
Wouldn't it be better to just change the polyfill in browser-index.ts to convert the return value back to a Uint8Array?

Reply:
I like that idea better :D